app/api/channel_api.py

```diff
@@ -0,0 +1,233 @@
+from fastapi import APIRouter, HTTPException
+from bson import ObjectId
+from app.database.schema import Channel
+from app.database.update_modal_channel import Change_channel
+from app.database.membership_modal import Membership
+from app.services.controller import (
+    fetch_channels,
+    create_channel,
+    remove_channel,
+    fetch_user_membership,
+    fetch_channel,
+    user_leave,
+    update_channel,
+    join_user,
+)
+
+from app.config.err_codes import Code
+# V1 add_admin,
+from app.services.error_custom import returnExceptions
+
+channels_router = APIRouter()
+
+# channel endpoints
+@channels_router.get("/channels/all")
+async def get_channels():
+    """
+    Endpoint to get all the channels
+
+    Raises:
+        HTTPException: ["All channels are private",
+                        or "Oops! Unexpected mongodb error occurred"]
```

Comment:
1. remove the square brackets
2. this doesnt explain much. What does "All channels are private" mean? when is it raised? same goes for "Oops! Unexpected mongodb error occured". Plus, don't put the error messages you will send as docs.